certify-sunbird-insurance.properties

```diff
@@ -9,108 +9,47 @@ mosip.certify.integration.audit-plugin=LoggerAuditService
-mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-get-url=${mosip.sunbird.url}/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.template-url=${spring_config_url_env}/${spring.application.name}/${active_profile_env}/${spring_config_label_env}/life-insurance-credential.vm
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-get-url=http://demo-registry.sunbird-registry:8081/api/v1/Insurance/
 mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-id=did:schema:e1f8df1d-c46c-483c-a882-3fc823666b9f
 mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-version=1.0.0
-mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-search-url=${mosip.sunbird.url}/api/v1/Insurance/search
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-search-url=http://demo-registry.sunbird-registry:8081/api/v1/Insurance/search
 
 ##Insurance Credential Configurations
 mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.static-value-map.issuerId=did:web:api.dev1.mosip.net:identity-service:8ebda1d0-665b-4bb7-abc7-d4bf56b6ee09
-mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/insurance-credential.vm
-mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-get-url=${mosip.sunbird.url}/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.template-url=${spring_config_url_env}/${spring.application.name}/${active_profile_env}/${spring_config_label_env}/insurance-credential.vm
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-get-url=http://demo-registry.sunbird-registry:8081/api/v1/Insurance/
 mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.cred-schema-id=did:schema:e1f8df1d-c46c-483c-a882-3fc823666b9f
 mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.cred-schema-version=1.0.0
-mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-search-url=${mosip.sunbird.url}/api/v1/Insurance/search
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-search-url=http://demo-registry.sunbird-registry:8081/api/v1/Insurance/search
 
 ## ------------------------------------------- UseCase specific default overriding properties ------------------------------------------------------------
 
 mosip.certify.domain.url=https://${mosip.injicertify.insurance.host}
 mosip.certify.identifier=${mosip.certify.domain.url}
 mosip.certify.authorization.url=https://${mosip.esignet.insurance.host}
 mosip.certify.database.name=inji_certify_insurance
-
+## Path to the json file containing svg template
+mosip.certify.svg-templates=
```

Comment:
Do we need this change in the sunbird plugin file ? I thought there are no changes to existing plugins.

Reply:
This property should be defined as it is being used by certify service. The service may break if this property is not added in config.

Reply:
But certify-default will have this, is that not sufficient ?

Reply:
Removing this property from sunbird and mosipid as it is not needed now